core/src/version.rc

```diff
@@ -23,7 +23,7 @@
         VALUE "InternalName", "Keyman Core"
         VALUE "LegalCopyright", "© SIL International"
         VALUE "LegalTrademarks", ""
-        VALUE "OriginalFilename", "KMNKBP0-0.DLL"
+        VALUE "OriginalFilename", "KMNCORE0-0.DLL"
```

Comment:
```suggestion
        VALUE "OriginalFilename", "KEYMANCORE0-0.DLL"
```

I also wonder about the need to append 0-0, surely we should be using the actual version?

Reply:
The next PR (#9793) updates the version number. But I don't think we want to use 17 as version number since this is the Core API version which usually doesn't change in breaking ways between Keyman versions.

Reply:
True. We just need to have a documented version number for the API then

Reply:
Good point. Created #9801.